Modules/Control/Field_To_Current.py

```diff
@@ -0,0 +1,9 @@
+import math
+AMOUNT_OF_WRAPS = 100
+RADIUS = 0.09
+
+def Field_To_Current(field):
+    return BaseException/(AMOUNT_OF_WRAPS*math.pi*RADIUS**2)
+
+def Calc_Field(current):
+    return current*AMOUNT_OF_WRAPS*math.pi*RADIUS**2
```

Comment:
**OPERATOR SPACING** and this is definitely not the first time I'm asking for it. `current * AMOUNT_OF_WRAPS * math.pi * RADIUS ** 2`

And apply this comment to everywhere else in this file. Operators spacing.